Documentation/gitsubmodules.txt

```diff
@@ -3,7 +3,7 @@ gitsubmodules(7)
 
 NAME
 ----
-gitsubmodules - mounting one repository inside another
+gitsubmodules - Mounting one repository inside another
```

Comment:
This change in the capitalization of the word "mounting" does not seem to be covered by the commit message. Is it an inadvertent change?

Reply:
It's not an inadvertent change ; if you check the diff for commit 1b81d8c, which taught "help" to read the guide list from "command-list.txt", all first words of the description of each guide were capitalized at the same time, so I did the same for gitsubmodules.

Reply:
Okay, could you say so in the commit message? This will help other reviewers not to stumble over this in the same manner that I did...

Reply:
yes sure!

Reply:
Excellent, thank you so much!